Cannot add more than 1 Column to Table

### Bug report

- __Package version(s)__: blueprint ver `1.7.1` and blueprint table ver `1.14.0` w/ typescript `2.3.3`
- __Browser and OS versions__: Chrome

#### Steps to reproduce

1. Create a `*.tsx` file with the following:

```ts
import * as React from 'react';
import { Table, Column } from '@blueprintjs/table';

import 'normalize.css';
import '@blueprintjs/core/dist/blueprint.css';
import '@blueprintjs/table/dist/table.css';


export class TableView extends React.Component<any, any> {
    render() {
        return (
            <Table numRows={20}>
                <Column />
                <Column />
            </Table>
        );
    }
}
```

#### Actual behavior

A nice typescript error:

```
Table.tsx
(12,20): error TS2322: Type '{ numRows: 20; children: Element[]; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Table> & Readonly<{ children?: ReactNode; }> & Rea...'.
  Type '{ numRows: 20; children: Element[]; }' is not assignable to type 'Readonly<ITableProps>'.
    Types of property 'children' are incompatible.
      Type 'Element[]' is not assignable to type 'ReactElement<IColumnProps> | undefined'.
        Type 'Element[]' is not assignable to type 'ReactElement<IColumnProps>'.
          Property 'type' is missing in type 'Element[]'.
```

#### Expected behavior

Compiles correctly.  The following code works:

```
import * as React from 'react';
import { Table, Column } from '@blueprintjs/table';

import 'normalize.css';
import '@blueprintjs/core/dist/blueprint.css';
import '@blueprintjs/table/dist/table.css';


export class TableView extends React.Component<any, any> {
    render() {
        return (
            <Table numRows={20}>
                <Column />
            </Table>
        );
    }
}
```

--- a/packages/table/src/table.tsx
+++ b/packages/table/src/table.tsx
@@ -61,7 +61,7 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
      * The children of a `Table` component, which must be React elements
      * that use `IColumnProps`.
      */
-    children?: React.ReactElement<IColumnProps>;
+    children?: React.ReactElement<IColumnProps> | React.ReactElement<IColumnProps>[];
 
     /**
      * If `true`, empty space in the table container will be filled with empty

